index.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/085384e239.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    
+    <!----------navbar setion------------------->
+    <nav class="nav-container">
+        <a href="">John Doe</a>
+        <ul class="items-menu">
+            <a href="#">
+                <li>Portfolio</li>
+            </a>
+            <a href="#">
+                <li>About</li>
+            </a>
+            <a href="#">
+                <li>Contact</li>
+            </a>
+            <a href="#">
+                <li><i class="fa-solid fa-envelope"></i></li>
+            </a>
+
+        </ul>
+        <a href="#">
+            <li><i class="fa-solid fa-bars"></i></li>
+        </a>
+
+    </nav>
+    <!--------------Hero section------------------------------------------->
+    <div class="hero-section">
+        <div class="hero-row">
+            <div class="hero-left">
+                <a href="#"><i class="fa-brands fa-github"></i></a>
+                <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+                <a href="#"><i class="fa-solid fa-m"></i></a>
+            </div>
```

Comment:
- [ ] Kindly make sure the social media links container is wrapped into the `ul` tag and each of the is wrapped into the `li` tag in order to structure your HTML semantically correctly.